releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -111,7 +111,7 @@ public void testGenerateOnlyMisc() throws Exception {
             createNotes(null, null, null, createAllNotes(), null), createBaseCliOptions()
                 .setGenerateAll(true).build());
 
-        Assert.assertEquals("no errors", 0, errors.size());
+        Assert.assertEquals("one error", 1, errors.size());
```

Comment:
Can we detach validation from GenerateAll ?
I still think it is very separate mode of execution.

Reply:
So all existing tests will work as before and new tests will do .setvalidate().build())

Reply:
Done.